client/src/components/Home/ActivityStream.js

```diff
@@ -0,0 +1,92 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import Composer from 'components/Chat'
+import Activity from './Activity'
+import T from 'components/T'
+import { defer } from 'lodash'
+
+import styles from './styles.module.scss'
+
+const ActivityStream = ({ activities, openModal }) => {
+  const [focusChat, setFocusChat] = React.useState(false);
+  const [scrolledToBottom, setScrolledToBottom] = React.useState(true);
+  const messageStream = React.useRef(null);
+  const activitiesList = React.useRef(null);
+
+  React.useEffect(() => {
```

Comment:
Looks like there are still some changes here outside of reorganizing the modules. Possible to apply these changes in a separate PR?

Reply:
You mean migrating to functional component ?

Reply:
See #145

Reply:
Rebased after #145 merge.